config.m4

```diff
@@ -36,13 +36,15 @@ if test "$PHP_PDO_CUBRID" != "no"; then
         if test "$ac_cv_sizeof_int_p" = "8"; then
     	    AC_MSG_NOTICE([Build static cci lib 64 bits])
             pushd $CCISRC_DIR
+            touch configure.ac
             chmod +x configure
             ./configure --enable-64bit
     	    make
             popd
         else
     	    AC_MSG_NOTICE([Build static cci lib])
             pushd $CCISRC_DIR
+            touch configure.ac
             chmod +x configure
```

Comment:
Even **chmod** may not change the last update date, it is desirable to change the order.
```
chmod +x configure
touch configure.ac
```